workshops/README.md

```diff
@@ -24,42 +30,50 @@ You'll start off by building your first website and will eventually make your wa
+| [Chat][chat]                               | JavaScript & Firebase                    | A real-time chat application                                                     | Hack Club staff                                                                                                        |
+| [Cringe 101][cringe_101]                   | HTML & CSS                               | Cringe 101 will give you the tools to create the most cringiest website ever.    | [@nguyenbrian][nguyenbrian], [@JevinSidhu][JevinSidhu], [@uditdesai][uditdesai], and [@vaibhavyadaram][vaibhavyadaram] |
+| [Find Bigfoot][find_bigfoot]               | HTML & CSS, JavaScript                   | A really simple game about finding Bigfoot                                       | [@ad510][ad510]                                                                                                        |
+| [Maze][maze]                               | JavaScript                               | A "Scary Maze Game" clone                                                        | Hack Club staff                                                                                                        |
+| [Soccer][soccer]                           | JavaScript                               | A simple  soccer game                                                            | Hack Club staff                                                                                                        |
+| [Swiper][swiper]                           | Swift (requires OS X)                    | Make a fun and addicting iOS game using Swift and SpriteKit                      | [@zach-cmiel][zachcmiel]                                                                                               |
+| [Thugify][thugify]                         | JavaScript                               | Learn to dynamically modify webpages with jQuery                                 | [@nguyenbrian][nguyenbrian], [@JevinSidhu][JevinSidhu], [@uditdesai][uditdesai], and [@vaibhavyadaram][vaibhavyadaram] |
+| [Twilio][twilio]                           | JavaScript (only works in North America) | A website that makes phone calls and texts                                       | Hack Club staff                                                                                                        |
+| [Twine][twine]                             | Twine (web interface)                    | Build a text adventure game                                                      | [@remixz][remixz]                                                                                                      |
+
+### Deprecated Workshops
+
+_:warning: These workshops are deprecated. Here be dragons._
```

Comment:
Replace deprecated with "no longer maintained. They may contain errors and are not recommended for club use."